src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -46,6 +46,18 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
       "...", "=>"
   )
 
+  // for value expressions
+  val binaryValueOperators = Set(
+    "+", "-", "*", "/", "**", "%",
+    "<<", ">>>", ">>", "&", "|", "^"
```

Comment:
These 2 lines should be indented with 4 spaces.

Reply:
And given my comments below, consider just including those directly inside `lexical.delimiters ++=` above.